xwiki-platform-core/xwiki-platform-ckeditor/xwiki-platform-ckeditor-ui/src/main/resources/CKEditor/VelocityMacros.xml

```diff
@@ -115,14 +116,19 @@
     ## editor is used.
     #set ($macro.previousDisplayMode = 'edit')
   #end
+  #if ("$!source.restricted" == 'true')
+    #set ($restricted = true)
+  #else
+    #set ($restricted = false)
+  #end
```

Comment:
```suggestion
  #set ($restricted = "$!source.restricted" == 'true')
```

Reply:
Hmm, I thought this did not work in Velocity, but it might be old Velocity 1.7 reflexes.